databases/week3/Additional_queries.sql

```diff
@@ -0,0 +1,44 @@
+-- Additional Quries
+-- Get meals that has a price smaller than a specific price fx 90
+select * from meal where price <90;
+
+-- Get meals that partially match a title
+select * from meal where title like '%butter%'; 
+
+-- Get meals that still has available reservations
+SELECT meal.id as meal_id, title, discription, location, price, max_reservations, 
+	COALESCE(SUM(number_of_guests), 0) AS total_reservations, 
+    COALESCE((max_reservations - SUM(number_of_guests)), max_reservations) as available_spots
+FROM meal
+LEFT JOIN reservation
+	ON meal.id = meal_id
+GROUP BY meal.id
+HAVING  max_reservations > total_reservations is null
+order by available_spots;
+
+-- Get meals that has been created between two dates
+select * from meal 
+where created_date between '2020-01-01' and '2020-12-01';
+
+-- Get only specific number of meals fx return only 5 meals
+select * from meal 
+where id between 1 and 5;
+
+-- Get the meals that have good reviews
+select meal.title , review.stars
+from meal inner join review on meal.id = review.meal_id
+where stars >3 order by stars desc;
```

Comment:
In this query you're still getting repeating meals if a meal has more than one review. I would use the `AVG(review.stars)` and group by `meal.id` to avoid that :)

Reply:
Thanks for the feedback I have changed these queries :)